TreeView/TreeViewBase.cs

```diff
@@ -80,6 +80,14 @@ public partial class TreeViewBase<T> : ComponentBase
         [Parameter]
         public bool InitiallyCollapsed { get; set; }
 
+        /// <summary>
+        /// Refresh the TreeView, including the root node
+        /// </summary>
+        public void Refresh()
+        {
+            StateHasChanged();

```

Comment:
It is unclear to me why this call to `StateHasChanged` is necessary. Isn't that implicit when the parent component changes (i.e. then updates the parameters to the TreeView component)?

Reply:
Sorry, it is my mistake. A try I made that I did not remove.